File: iniciante/python/1080-maior-e-posicao.py
def maior_posicao():
    posicao = maior = 0
    for i in range(1, 101):
        numero = int(input())
        if numero > maior:
            maior = numero
            posicao = i
    print(f'{maior}\n'
          f'{posicao}')


maior_posicao()

File: iniciante/c/1080-maior-e-posicao.c
#include <stdio.h>

int main(){
    int i, valor, maior = 0, posicao = 0;
    for (i = 0; i < 100; i++){
        scanf("%d", &valor);
        if (valor > maior){
            maior = valor;
            posicao = i + 1;
        }
    }
    printf("%d\n", maior);
    printf("%d\n", posicao);
    return 0;
}
